easybuild/easyconfigs/b/BRiAl/BRiAl-1.2.12-GCC-13.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'BRiAl'
+version = '1.2.12'
+
+homepage = 'https://github.com/BRiAl/BRiAl'
+description = """BRiAl is the legacy version of PolyBoRi maintained by sagemath developers."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/BRiAl/BRiAl/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['ca009e3722dd3f0a60d15501caed1413146c80abced57423e32ae0116f407494']
+
+dependencies = [
+    ('Boost', '1.83.0'),
```

Comment:
@PetrKralCZ Are you sure that the `CUDD` and `Python` dependencies that are there in the existing `BRiAl-1.2.12-GCC-11.3.0.eb` are not required?

Would be strange, since there's no version bump, especially for `CUDD`.

Reply:
It seems like without the `CUDD` dependency, it's building its own copy of it. From the log:
```
Making all in cudd
```

Not sure why `Python` is included as a dependency though, maybe @migueldiascosta remembers (see #19208)

Reply:
@boegel I don't. Possibly I added it because there are also some python bindings, but those are built separately by sage itself, so no need for python here